roles/acm_hypershift/tasks/delete-cluster.yml

```diff
@@ -15,8 +15,23 @@
       --name={{ ah_cluster_generated_name }}
       --namespace={{ ah_clusters_ns }}
       --infra-id={{ ah_cluster_generated_name }}
-      --cluster-grace-period 0
   changed_when: false
   when:
-    - hosted_cluster.resources | length == 1
+    - hosted_cluster.resources | default([]) | length == 1
+
+- name: Confirm that HCP is deleted
```

Comment:
I haven't seen this task as tested in the last job, could we have the chance to test this `delete-cluster` play?

Reply:
yeah, just need to run it with a fixed cluster_name. The last jobs were run using kvirt as the name for the hcp cluster, So it will be deleted.